src/services/alert.service.js

```diff
@@ -1,59 +1,59 @@
 import Swal from 'sweetalert2'
 
-export class AlertService{
+export class AlertService {
 
-    static errorMessage(title = 'Error', text = 'Error') {
+  static errorMessage(title = 'Error', text = 'Error') {
     Swal.fire({
       icon: 'error',
       title: title,
       text: text
     })
   }
 
-  static successMessage(title = 'Success'){
+  static successMessage(title = 'Success', timer = 1500) {
     Swal.fire({
-        position: 'center',
-        icon: 'success',
-        title: title,
-        showConfirmButton: false,
-        timer: 1500
-      })
+      position: 'center',
+      icon: 'success',
+      title: title,
+      timer: timer,
+      showConfirmButton: false
+    })
   }
 
   static selectMessage(title = 'Are you sure?', confirmButtonText = 'Confirm',
-  denyButtonText = 'Cancel', confirmedText = 'Confirmed', DenyText = 'Deny') {
+    denyButtonText = 'Cancel', confirmedText = 'Confirmed', DenyText = 'Deny') {
     Swal.fire({
-        title: title,
-        showDenyButton: true,
-        showCancelButton: true,
-        confirmButtonText: confirmButtonText,
-        denyButtonText: denyButtonText,
-      }).then((result) => {
-        if (result.isConfirmed) {
-            this.successMessage(confirmedText)
-            return false
-        } else if (result.isDenied) {
-            Swal.fire(DenyText, '', 'info')
-          return true
-        }
-      })
+      title: title,
+      showDenyButton: true,
+      showCancelButton: true,
+      confirmButtonText: confirmButtonText,
+      denyButtonText: denyButtonText,
+    }).then((result) => {
+      if (result.isConfirmed) {
+        this.successMessage(confirmedText)
+        return false
+      } else if (result.isDenied) {
+        Swal.fire(DenyText, '', 'info')
+        return true
+      }
+    })
   }
 
   static confirmMessage(title = 'Are you sure?', text = 'Are you realy sure?',
```

Comment:
are we sure that this should be static ?

Reply:
without static methods in other classes are not available

Reply:
It's clear. You should inject the instance if you remove the static. 
The answer for this question "are we sure that this should be static ?" should be based on opinion with facts that describe the technical solution and provide the best practice in scope of creating wrapper methods for Swal with preset configuration.